EIPS/eip-7069.md

```diff
@@ -0,0 +1,145 @@
+---
+eip: 7069
+title: Revamped CALL instructions
+description: Introduce CALL2, DELEGATECALL2 and STATICCALL2 with simplified semantics
+author: Alex Beregszaszi (@axic), Paweł Bylica (@chfast), Danno Ferrin (@shemnon), Andrei Maiboroda (@gumb0)
+discussions-to: https://ethereum-magicians.org/t/eip-revamped-call-instructions/14432
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-05-05
+requires: 150, 211, 214, 2929
+---
+
+## Abstract
+
+Introduce three new call instructions, `CALL2`, `DELEGATECALL2` and `STATICCALL2`, with simplified semantics. The existing call instructions remain unchanged.
+
+The new instructions do not allow specifying a gas limit, but rather rely on the "63/64th rule" ([EIP-150](./eip-150.md)) to limit gas. An important improvement is the rules around the "stipend" are simplified, and callers do not need to perform special calculation whether the value is sent or not. 
+
+Furthermore, the obsolete functionality of specifying output buffer address is removed, because it is predominantly unused and implementers prefer to use `RETURNDATACOPY` instead. One exception is the case when the expected return size is known (i.e. non-dynamic return values), Solidity still uses the output buffer.
```

Comment:
but it isn't Rationale, but what the EIP does

Reply:
This would be better i think:

Furthermore, the obsolete functionality of specifying output buffer address is removed in favor of using `RETURNDATACOPY` instead.

Reply:
Moved.